src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/GeneratedPDFContentV2Test.java

```diff
@@ -44,30 +64,51 @@ private static String textContentOf(byte[] pdfData) throws IOException {
         }
     }
 
-    @SuppressWarnings("unchecked")
+    @After
+    public void verifyAuth() {
+        verify(authService).authenticate("some-service-auth-header");
```

Comment:
Verifications and assertions shouldn't be done in setup/teardown methods.

Reply:
shouldn't be, but simplest way to test requirement for all positive and negative tests as it must be covered